php-checks/src/main/java/org/sonar/php/checks/wordpress/WordPressDbRepairCheck.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.php.checks.wordpress;
+
+import java.util.Set;
+import org.sonar.check.Rule;
+import org.sonar.php.checks.utils.CheckUtils;
+import org.sonar.php.utils.collections.SetUtils;
+import org.sonar.plugins.php.api.tree.expression.FunctionCallTree;
+
+@Rule(key = "S6346")
+public class WordPressDbRepairCheck extends WordPressConfigVisitor {
+
+  private static final String MESSAGE = "Make sure allowing unauthenticated database repair is intended.";
+
+  @Override
+  protected Set<String> configsToVisit() {
+    return SetUtils.immutableSetOf("WP_ALLOW_REPAIR");
```

Comment:
Any reason not to use `Collections.singleton()` like we did in all the other checks?
I think we should prefer to use built-in functions when available for what we need.